tests/src/lib.rs

```diff
@@ -119,14 +119,35 @@ pub fn mk_compiler(
     #[cfg(cargo_profile = "release")]
     let mode = "release";
 
+    let prelink_args = env::var("PRELINK_PASSES").unwrap_or_default();
+    let postlink_args = env::var("POSTLINK_PASSES").unwrap_or_default(); 
+
+    let mut yk_config_args: Vec<&str> = Vec::new();
+    yk_config_args.push(mode);
+
+    if !prelink_args.is_empty() {
+        yk_config_args.push("--prelink-pipeline");
+        yk_config_args.push(&prelink_args);
+    }
+    yk_config_args.push("--cflags");
```

Comment:
I think lines 132, 133, 138, and 139 can be part of the initial vec creation? I don't think their order is important?

Reply:
so `--prelink-pipeline` and `--postlink-pipeline` should be passed before  `--cflags` and `--ldflags` respectively.

Reply:
OK.